pkg/api-server/config_ws_test.go

```diff
@@ -336,6 +336,10 @@ var _ = Describe("Config WS", func() {
               "generateZoneToken": {
                 "users": ["mesh-system:admin"],
                 "groups": ["mesh-system:admin"]
+              },
+              "getConfigDump": {
+                "users": ["mesh-system:anonymous","mesh-system:admin"],
```

Comment:
I think the list should be empty. I think it also should be empty for other accesses, but that's out of scope of this PR.